src/plone/formwidget/hcaptcha/nohcaptcha.py

```diff
@@ -93,17 +91,16 @@ def encode_if_necessary(s):
     params = parse.urlencode(
         {
             "secret": secret_key,
-            "remoteip": remoteip,
             "response": hcaptcha_response_field,
         }
     )
 
     request = Request(
-        url="https://{0}/recaptcha/api/siteverify".format(verify_server),
+        url="https://{0}/siteverify".format(verify_server),
         data=params,
         headers={
             "Content-type": "application/x-www-form-urlencoded",
-            "User-agent": "noReCAPTCHA Python",
+            "User-agent": "noHCAPTCHA Python",
```

Comment:
The "no" old user agent string "noReCAPTCHA" comes from the [`norecaptcha` package](https://pypi.org/project/norecaptcha/). I suggest removing the "no" from the new user agent string.